editors/code/src/debug.ts

```diff
@@ -66,6 +66,12 @@ export async function startDebugSession(ctx: Ctx, runnable: ra.Runnable): Promis
     return vscode.debug.startDebugging(undefined, debugConfig);
 }
 
+function createCommand(extensionId: string): string {
+    return `rust-analyzer.installExtension?${encodeURIComponent(
```

Comment:
Can we call `workbench.extensions.installExtension` directly instead of defining a `rust-analyzer.installExtension` which calls `workbench.extensions.installExtension`?

Reply:
I am working on this but it is not documented anywhere in their API. 🥲

Reply:
After some digging through their codebase, I found it, [extension.open](https://github.com/search?q=repo%3Amicrosoft%2Fvscode+extension.open&type=code).